tests/AcsiTswriterTest/FactoryTest.php

```diff
@@ -77,22 +73,21 @@ public function testRegistrationWithInstance()
         $this->assertArrayNotHasKey($class, $registry->getValue($this->factory));
     }
 
-    /**
-     * @expectedException \TS\Writer\Exception\FactoryException
-     */
-    public function testFactoryException()
+    public function testFactoryException(): void
     {
+        $this->expectException(FactoryException::class);
+
         $this->factory->createForType('txt');
     }
 
-    public function testFactory()
-    {
-        $instance = new Txt(new EventDispatcher());
-
-        $this->factory->registerWriter($instance);
-
-        $writer = $this->factory->createForType('txt');
-
-        $this->assertEquals($instance, $writer);
-    }
+//    public function testFactory(): void
+//    {
+//        $instance = new Txt(new EventDispatcher());
+//
+//        $this->factory->registerWriter($instance);
+//
+//        $writer = $this->factory->createForType('txt');
+//
+//        $this->assertEquals($instance, $writer);
+//    }
```

Comment:
Why commented code?

Reply:
oops ..thats not intentionally . may be added to test something and then forgot to update it.